Twitter/Data/Migrations/20230113151851_AddPropertyIsArchivedToPostModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Do you also need to add filter where you will check that archived posts will be ignored or that is part from another task?

Reply:
It is part from another task